pkgs/by-name/fr/freecad/package.nix

```diff
@@ -40,26 +40,27 @@
   nix-update-script,
 }:
 let
-  inherit (python311Packages)
+  pythonDeps = with python3Packages; [
```

Comment:
careful. You are mixing qt5 and qt6 here (shiboken2 / shiboken6).
The old version was an inherit, with `lib.optionals` in `buildInputs`. Either you mve the optional here, or you keep the `inherit` here and don't `++` in buildInputs. Currently a bunch of stuff is doubled, some of which in dangerous ways.